src/fiatExchanges/utils.tsx

```diff
@@ -259,12 +260,12 @@ export const filterLegacyMobileMoneyProviders = (
   providers: LegacyMobileMoneyProvider[] | undefined,
   flow: CICOFlow,
   userCountry: string | null,
-  selectedCurrency: CiCoCurrency
+  selectedCurrency: string
```

Comment:
looking at how this is used (only one use, from `SelectProvider`), we can prolly use `tokenId` here instead of symbol, if we use consts like `cUsdTokenId` as suggested in another comment.